src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const inCalculator = {
+    add(callback, number) {
+      callback.result += number;
+    },
+    subtract(callback, number) {
+      callback.result -= number;
+    },
+    multiply(callback, number) {
+      callback.result *= number;
+    },
+    divide(callback, number) {
+      callback.result /= number;
```

Comment:
interesting solution, but maybe you should add throw error (dividing zero)

Reply:
Agree, you can't really divide by zero

Reply:
ok